pkg/tsdb/cloudwatch/response_parser.go

```diff
@@ -87,6 +89,30 @@ func aggregateResponse(getMetricDataOutputs []*cloudwatch.GetMetricDataOutput) m
 	return responseByID
 }
 
+func parseLabels(cloudwatchLabel string, query *models.CloudWatchQuery) (string, data.Labels) {
+	dims := make([]string, 0, len(query.Dimensions))
+	for k := range query.Dimensions {
+		dims = append(dims, k)
+	}
+	sort.Strings(dims)
+
+	splitLabels := strings.Split(cloudwatchLabel, keySeparator)
+	labelsIndex := 1
```

Comment:
nit: could you add a note here for why `labelsIndex` starts at 1. It wasn't clear to me at first until I traced it back to `buildSearchExpressionLabel`.

Reply:
done!